src/server/handler.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs');
+const path = require('path');
+
+// Home page Handler
+
+const homeHandler = (request, response) => {
+	const htmlPath = path.join(__dirname, '..','..', 'public', 'index.html');
+	fs.readFile(htmlPath, (error, file) => {
+		if (error) {
+			errorHandler(request, response);
+			return;
+		}
+		response.writeHead(200, {'Content-Type': 'text/html'});
+		response.end(file);
+	});
+};
+
+//Public Handler
+
+const publicHandler = (request, response) => {
+	const extention = request.url.split('.')[1];
+	const ContentTypeMapping = {
+		html: 'text/html',
+		css: 'text/css',
+		js: 'application/js',
+		jpg: 'image/jpg',
+		png: 'image/png',
+		ico: 'image/x-ico',
+		jpeg: 'image/jpeg',
+	};
+
+	if (ContentTypeMapping[extention] === undefined) {
+		errorHandler(request, response);
+		return;
```

Comment:
Instead of returning on the second line you could just write `return errorHandler(request, response);`

Reply:
Done